### PR TITLE
Gradebook: Redirect to LP that contains the exercise in exercise link

### PR DESCRIPTION
**Pull request by AngelFQC**
Refs #2184

### CODE REVIEW

File: main/gradebook/exercise_jump.php [hunk 1]
```diff
@@ -22,40 +22,31 @@
...
-        'session_id' => $session_id,
-        'cidReq' => $cidReq,
-        'gradebook' => $gradebook,
-        'origin' => '',
-        'learnpath_id' => '',
-        'learnpath_item_id' => '',
-        'exerciseId' => intval($_GET['doexercise']),
-    ]);
-}
-
```
Comment (ywarnier) [inline main/gradebook/exercise_jump.php]: I need an explanation about why this section is gone.

File: main/gradebook/exercise_jump.php [hunk 2]
```diff
@@ -69,13 +60,32 @@
...
+                        'session_id' => $session_id,
+                        'cidReq' => $cidReq,
+                        'gradebook' => $gradebook,
+                        'origin' => '',
+                        'learnpath_id' => '',
+                        'learnpath_item_id' => '',
+                        'exerciseId' => (int) $_GET['exerciseId'],
+                    ]
+                );
             }
```
Comment (AngelFQC) [inline main/gradebook/exercise_jump.php]: It has been moved here

**Comment by AngelFQC** [comment]
Move to master branch

**Comment by AngelFQC** [comment]
Commits applied on master branch.

**Pull request merged**